docs/tutorials/simple_trading.py.md

```diff
@@ -84,11 +84,11 @@ class SimpleMarketListener(MarketStreamListener):
     futures = [instrument for instrument in instrument_data['data'].values() if instrument['type'] == 'futures'][0]
     selected_futures_id = futures['instrument_id']
 
-  def on_order_book(self, order_book):
+  def on_order_book(self, order_book):  
     if order_book['instrument_id'] != selected_futures_id:
       return 
     bids = order_book['bids']
-    if bids and (not bids[0] or bids[0][0] > sell_threshold):
+    if bids and (not bids[0] or float(bids[0][0]) > sell_threshold):
```

Comment:
It'd be good to guarantee that bids are present and that values are properly typed.

Then:
```python
if bids[0] and bids[0][0] > sell_threshold:
```

Also, what do we do with big decimals? I think we may want to enable adding converter middlewares to streams, so that everyone can convert strings to implementation of liking (and provide middlewares for most popular big decimal implementations).